src/main/frontend/components/search.cljs

```diff
@@ -211,8 +211,14 @@
                                  :block
                                  block))
 
-                              nil)
-                            (search-handler/clear-search!))
+                              :new-page
+                              (page-handler/create! search-q)
+
+                              :file
```

Comment:
`:new-page` and `:file` will be handled in the same logic of normal `on-chosen`

Reply:
Awesome